css/index.css

```diff
@@ -0,0 +1,162 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+.card__container {
+  margin: 0 auto;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  padding: 36px 16px 16px;
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  -webkit-box-pack: start;
+      -ms-flex-pack: start;
+          justify-content: start;
+}
+.card__image {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+.card__title {
+  margin-top: 0;
+  margin-bottom: 4px;
+  text-align: start;
+  text-wrap: nowrap;
```

Comment:
The `text-wrap` property is not a valid CSS property. You might want to use `white-space: nowrap;` if you intend to prevent text from wrapping.